src/components/Contact.jsx

```diff
@@ -1,124 +1,117 @@
-            </div>
-            <div className="col-md-6">
-              <form
-                className="needs-validation"
-                noValidate
-                onSubmit={this.submitForm}
-                action="https://formspree.io/xzbjwkae"
-                method="POST"
-              >
-                <div className="form-group mb-4">
-                  <label className="h5 text-dark contact-label">Name</label>
-                  <input
-                    type="name"
-                    className="form-control"
-                    name="name"
-                    placeholder="Eg. Harry Singh"
-                    required
-                  />
-                </div>
-                <div className="form-group mb-4">
-                  <label className="h5 contact-label">Email</label>
-                  <input
-                    type="email"
-                    className="form-control"
-                    name="email"
-                    placeholder="Eg. example@example.com"
-                    required
-                  />
-                </div>
-                <div className="form-group mb-4">
-                  <label className="h5 contact-label">Subject</label>
-                  <input
-                    type="text"
-                    className="form-control"
-                    name="subject"
-                    placeholder="Write the subject of the mail here."
-                    required
-                  />
+        <div>
+            <h1 className="text-center">Contact</h1>
+            <div
+                className="container-fluid rounded shadow-lg py-4 px-3 mt-5"
+                style={{width: "70%"}}
+            >
+                <div className="row mt-2">
+                    <div className="col-md-6">
+                        <img
+                            src={contact_illustration}
+                            alt="contact_illustration"
+                            className="float-left"
+                            style={{width: "100%", height: "100%"}}
+                        />
+                    </div>
+                    <div className="col-md-6">
+                        <form
+                            className="needs-validation"
+                            noValidate
+                            onSubmit={(ev) => submitForm(ev)}
+                            action="https://formspree.io/xzbjwkae"
+                            method="POST"
+                        >
+                            <div className="form-group mb-4">
+                                <label className="h5 text-dark contact-label">Name</label>
+                                <input
+                                    type="name"
+                                    className="form-control"
+                                    name="name"
+                                    placeholder="Eg. Harry Singh"
+                                    required
+                                />
```

Comment:
Use state to hold all form inputs using OnChange and use that state to show the value in all.
Make new state of all form inputs

Reply:
Ok.